components/alert/index.tsx

```diff
@@ -114,7 +114,7 @@ const Alert: CompoundedComponent = ({
   closeIcon = <CloseOutlined />,
   action,
   ...props
-}) => {
+}: AlertProps & CompoundedComponent) => {
```

Comment:
This is return type. Not props should not merge into AlertProps.

Reply:
My fault. I would modify it!!

Reply:
@guan404ming wait! Pls check https://github.com/ant-design/ant-design/pull/41900#issuecomment-1515626474 to avoid unnecessary effort